spec/features/add_todos_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.feature "Todo List", type: :feature do
+  it "enters and displays a new todo item" do
+    visit '/'
+    page.fill_in 'new-todo', with: 'Buy milk'
+    click_button('add')
+    expect(page).to have_text('Buy milk')
+  end
+
+  it "enters and displays a new todo item" do
+    visit '/'
+    page.fill_in 'new-todo', with: 'Buy cereal'
+    click_button('add')
+    expect(page).to have_text('Buy cereal')
+  end
+
```

Comment:
Feature specs are fairly heavy and add run times to your suite. It's great to have them for testing that things work as you'd expect, but I'd avoid duplicating them as you have here. In general, write your feature specs to check a family of similar behaviors instead of minor variations.

There's no difference between your first and second spec, besides the particular string you're evaluating. The difference between your second and third is that you're passing an integer instead. That's better suited to your unit test (in the model folder). If your docstring (the part in the `it` block) doesn't change between the tests, that's a good indication that your test might be redundant.